deploy/V2.ts

```diff
@@ -1,28 +1,132 @@
 import { HardhatRuntimeEnvironment } from "hardhat/types";
```

Comment:
we should have left this deployment alone and create a new one just for the masterchef V2.1 update